<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QDialog" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start Menu</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Player Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Go!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioRedBaronButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>111</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>Red Baron</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBomberButton">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>80</y>
     <width>111</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>Bomber</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioThunderbirdButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>111</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>Thunderbird</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
